Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -31,8 +31,9 @@
   </ItemGroup>
   
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.2'">
+    <PackageReference Include="Microsoft.NetCore.App" Version="2.2.8" />

```

Comment:
_CVE Initiated library update - net.core.app2.2.0 nupkg
This package needs to be updated from 2.2.0 to 2.2.1_

Updating to 2.2.8 instead of 2.2.1 because 2.2.8 is the latest stable version